code/modules/antagonists/ninja/ninja.dm

```diff
@@ -136,23 +110,14 @@
 
 /datum/antagonist/ninja/admin_add(datum/mind/new_owner,mob/admin)
 	var/adj
-	switch(input("What kind of ninja?", "Ninja") as null|anything in list("Random","Syndicate","Nanotrasen","No objectives"))
+	switch(input("What kind of ninja?", "Ninja") as null|anything in list("Random","No objectives"))
```

Comment:
This doesn't need to exist
Traitors already init with objectives

Reply:
This is only under admin_add, so an admin can create an objectiveless ninja as a blank slate to then add custom objectives to. 

This will only be utilized from specifically transforming a chosen player into a ninja, not part of the RNG system at all.

Reply:
While I don't like friendly ninjas existing, I don't see why generating a ninja without objectives as an admin tool should be removed

🤔

Reply:
Other antagonists always init with objectives.
Not sure why initializing ninja specifically without objectives is needed.

Reply:
discussed in discord, removing for now - will push after testing.